### PR TITLE
Move typography defaults into `tailwind.css` file

### LINKED ISSUE #1094: Add font defaults to tailwind css file
Currently the default font color and font weight for headings is inherited from compiled sass files. We should set this in the tailwind css file instead, in the base layer, so we can delete the sass files later on once we know it is safe to do so.

```
  html {
    @apply text-dark-charcoal;
  }

  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    @apply font-bold;
  }
```

I will do this, because it's super quick 😄

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #1094 by @zackkrida

## Description

Moves the default values for typography into the main tailwind.css file and deletes the corresponding sass. If this is working correctly, the site should look identical everywhere.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VAudioTrack/layouts/VBoxLayout.vue [hunk 1]
```diff
@@ -6,9 +6,11 @@
     >
       <div class="absolute inset-0 flex flex-col">
         <div class="info flex-grow flex flex-col justify-between p-4">
-          <span class="font-heading font-semibold leading-snug line-clamp-3">{{
-            audio.title
-          }}</span>
+          <h2
+            class="font-heading font-semibold leading-snug line-clamp-3 text-base"
+          >
```
Comment (zackkrida) [inline src/components/VAudioTrack/layouts/VBoxLayout.vue:11]: This is an unrelated improvement, apologies! Adding `text-base` _was_ necessary to keep these looking correct, however.

File: src/styles/tailwind.css [hunk 2]
```diff
@@ -3,7 +3,26 @@
...
 
 @layer base {
+  html {
+    @apply text-dark-charcoal m-0 font-normal leading-loose;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 400;
+    margin: 0;
+    font-display: swap;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```
Comment (zackkrida) [inline src/styles/tailwind.css:13]: this is odd, but it _does_ make text look much better in Firefox on OSX. This is usually the default rendering Firefox uses, but I think it gets confused on the Openverse yellow.

File: src/styles/tailwind.css [hunk 3]
```diff
@@ -3,7 +3,26 @@
 @tailwind utilities;
 
 @layer base {
+  html {
+    @apply text-dark-charcoal m-0 font-normal leading-loose;
+    -moz-osx-font-smoothing: grayscale;
```
Comment (dhruvkb) [inline src/styles/tailwind.css:8]: This is defined twice.

File: src/styles/tailwind.css [hunk 4]
```diff
@@ -3,7 +3,26 @@
...
+    font-weight: 400;
+    margin: 0;
```
Comment (dhruvkb) [inline src/styles/tailwind.css:10]: Could be replaced with `@apply font-normal m-0`.

File: src/styles/tailwind.css (hunk 2 above)
Comment (dhruvkb) [reply src/styles/tailwind.css:13]: This is cool, how did you identify this? 

Side note: Tailwind has [font-smoothing utilities](https://tailwindcss.com/docs/font-smoothing#applying-font-smoothing) but they wouldn't allow values to be browser-specific.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Watching SCSS files shrink is always a pleasure. I have a couple of notes though.

File: src/styles/tailwind.css (hunk 3 above)
Comment (zackkrida) [inline src/styles/tailwind.css:8]:
```suggestion
```

File: src/styles/tailwind.css (hunk 4 above)
Comment (zackkrida) [inline src/styles/tailwind.css:10]:
```suggestion
```

File: src/styles/tailwind.css (hunk 2 above)
Comment (zackkrida) [reply src/styles/tailwind.css:13]: > This is cool, how did you identify this?

It was already set, and I'm honestly not sure when or why 😓 but removing it quite noticeably makes the font inter artificially thicker in Firefox.

**Comment by zackkrida** [comment]
thanks for the cleanup suggestions @dhruvkb, i actually had most styles duplicated as an `@apply` and plain css.

**Review by dhruvkb — APPROVED** [review]

**Review by sarayourfriend — APPROVED** [review]
LGTM!
